airbyte-integrations/connectors/destination-dev-null/src/main/resources/spec.json

```diff
@@ -1,179 +1,190 @@
 {
   "documentationUrl": "https://docs.airbyte.com/integrations/destinations/e2e-test",
```

Comment:
@edgao @stephane-airbyte there was a lot of rebasing, I probably picked this up by accident

this should be deleted in the new world, correct? `expected-spec.json` is for the test, correct?

Reply:
(obviously I'll need to make this test both specs)

Reply:
yep, expected-spec is the new thing that tests rely on. My understanding is that at runtime, the new cdkwill generate the spec from code, right? I.e. in the old connector, this spec.json file was actually the source of truth for spec - if that's no longer true, then we can kill it :hocho: